plugins/modules/ipadnszone.py

```diff
@@ -352,6 +359,24 @@ def get_ipa_skip_nameserver_check(self):
         if not self.zone and self.ipa_params.skip_nameserver_check is not None:
             return self.ipa_params.skip_nameserver_check
 
+    @staticmethod
+    def _reverse_zone_name(ipaddress):
+        reverse_ip = dns.reversename.from_address(ipaddress)
+        try:
+            reverse_zone = dns.resolver.zone_for_name(reverse_ip)
+        except dns.resolver.NoNameservers:
+            # if reverse zone cannot be resolved, we'll use a sensible default.
```

Comment:
Is that what the CLI current do? If so I'd add that to the comment as well.

Reply:
That is similar to what CLI does. CLI does that with a slightly different approach, using netaddr instead of dnspython, but the effect is the same. I'll add a comment to the function and to the commit message.

Reply:
It is similar to what the CLI does, but the CLI requires confirmation from the user. We can´t. All we can do is inform the user which zone was created.